File: 02_LOAD_RASTER.R
# Install the terra package if not already installed
if (!requireNamespace("terra", quietly = TRUE)) {
  install.packages("terra")
}

# Load the terra library
library(terra)

# Define the path to the RGB raster TIFF file
raster_path <- "C:/Users/alvarece/Desktop/MODELAMIENTO Y PROGRAMACION GIS/PRACTICAS/DATOS/BermejoBS09_Ortomosaico.tif"

# Load the raster as a multi-band raster
rgb_raster <- rast(raster_path)

# Check the raster properties
print(rgb_raster)

# Plot the RGB raster using the first three bands as Red, Green, Blue
plotRGB(rgb_raster, r = 1, g = 2, b = 3, stretch = "lin")

File: 02_LOAD_RASTER/02_LOAD_RASTER.py
# Import the arcpy library
import arcpy

# Define the path to the raster TIFF file
raster_path = r"C:/Users/alvarece/Desktop/MODELAMIENTO Y PROGRAMACION GIS/PRACTICAS/DATOS/BermejoBS09_Ortomosaico.tif"

# Set the current ArcGIS Pro project and map
aprx = arcpy.mp.ArcGISProject("CURRENT")  # Use the currently open project
map_obj = aprx.activeMap  # Get the active map

# Directly add the raster to the map without creating an intermediate layer
map_obj.addDataFromPath(raster_path)

# Save the changes to the project
aprx.save()

print("Raster loaded and added to the map successfully.")
